### PR TITLE
Removes mentions of Russians, Capitalism/Stalinium Golems, Derelict Drone Subtype, Tweaks AI Ion Laws

### PR DESCRIPTION
**Pull request by meemofcourse**
## About The Pull Request

It does exactly what it says on the tin.

## Why It's Good For The Game

Not lore compliant. Old sprites with gear that nobody or nothing uses. "Gas the lizards" is not an acceptable term.

## Changelog

:cl:
del: Russian Battle Coat, Russian Uniform, Russian Officer Uniform, Russian Helmet, Russian Boots, Russian Balaclava
imagedel: Deleted sprites relating to the outfits above.
del: Stalinium and Capitalist Golems
del: Derelict Drones
tweak: The butterbear is now on the Frontiersmen team.
tweak: Modified the list of laws that an AI may recieve from being ioned.
sounddel: The Soviet Anthem and the US Anthem.
/:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
Hello, I like money!

File: strings/ion_laws.json [hunk 1]
```diff
@@ -971,27 +934,18 @@
...
 		"DRUGS",
-		"MEXICANS",
-		"CANADIANS",
-		"RUSSIANS",
 		"HULKS",
 		"SLIMES",
 		"SKELETONS",
-		"CAPITALISTS",
 		"SINGULARITIES",
 		"ANGRY BLACK MEN",
```
Comment (Zevotech) [inline strings/ion_laws.json]: this one too just to be safe?

**Review by Imaginos16 — APPROVED** [review]
Gonna need at least one testmerge but the PR seems fine otherwise

**Comment by Imaginos16** [comment]
Holy space clearing, batman!

image

**Pull request merged**